cookbook/attributes/agent.rb

```diff
@@ -0,0 +1,118 @@
+default[:baragon][:agent_log] = '/var/log/baragon/baragon_agent.log'
+
+default[:baragon][:agent_yaml] = {
+  'server' => {
+    'type' => 'simple',
+    'applicationContextPath' => '/baragon-agent/v2',
+    'connector' => {
+      'type' => 'http',
+      'port' => 8882
+    }
+  },
+  'zookeeper' => {
+    'sessionTimeoutMillis' => 60_000,
+    'connectTimeoutMillis' => 5000,
+    'retryBaseSleepTimeMilliseconds' => 1_000,
+    'retryMaxTries' => 3
+  },
+  'loadBalancerConfig' => {
+    'name' => 'default',
+    'domain' => 'vagrant.baragon.biz',
+    'rootPath' => '/tmp'
+  }
+}
+
+if node[:nginx]
```

Comment:
@ssalinas how do you feel about this method of handling nginx-related attributes?